18F/identity-idp
# Update alerts in mailers to align to match design system

## 🛠 Summary of changes

Updates alert content in mailers to align to the new design system styling introduced in https://github.com/18F/identity-design-system/pull/439 .

## 📜 Testing Plan

1. Go to http://localhost:3000/rails/mailers/user_mailer/in_person_ready_to_verify
2. Observe alert appears visually consistent with other alerts in application ([example](https://idp.dev.identitysandbox.gov/components/inspect/alert/preview))

Extra:
- To simulate outage alert visible, you can change condition [on this line](https://github.com/18F/identity-idp/blob/6939515a88455f292be1288721c6d468318873a3/app/views/user_mailer/shared/_in_person_ready_to_verify.html.erb#L1) to always be `true` (e.g. `|| true`)

## 👀 Screenshots

Before|After
---|---
![image](https://github.com/user-attachments/assets/99287c7e-2c12-4f07-9347-aa15c8764c65)|![image](https://github.com/user-attachments/assets/20bbf580-cc62-409e-80ca-9c368e5563c9)
